taca/storage/storage.py

```diff
@@ -114,3 +218,31 @@ def _send_to_swestore(f, dest, remove=True):
         else:
             LOG.info("Run {} is not {} days old yet. Not archiving".format(run, str(days)))
     os.remove("{}.archiving".format(run.split('.')[0]))
+
+
+def get_closed_projects(projs,pj_con,days,logger=None):
```

Comment:
I wanted to LOG some stuff if necessary, but since `LOG` is initiated within different methods, I was not sure if it is accessible in this method. I mean would the following example work ?

```
def do_something(params):
    LOG = logger()
    do_another_thing(whatever)


def do_another_thing(params):
    print params
    LOG.info('will this work ?')
```

Will this execute without error or would it fail saying `LOG` global name not defined ? I always have this doubt if a `name` is defined in parent method where another method is called, and can the `name` be accessed within the child method without having to pass ?